tests/test_http_response.py

```diff
@@ -699,6 +714,30 @@ def test_html5_meta_charset(self):
         self._assert_response_values(r1, 'gb2312', body)
 
 
+class JsonResponseTest(TextResponseTest):
+
+    response_class = JsonResponse
+
+    def test_js_encoding(self):
+        body = b"[{\"id\": \"1234\", \"type\": \"event\"}]"
```

Comment:
```suggestion
        body = """[{"id": "1234", "type": "event"}]"""
```
A little bit cleaner this way